microsoft-edge/webview2/howto/chromium-devtools-protocol.md

```diff
@@ -0,0 +1,190 @@
+       Display the geolocation coordinates of the user in Microsoft Edge  
+    :::image-end:::  
+    
+## Step 2: Display geolocater.html in a WebView2  
+
+1.  To create a WebView2 app, use either of the following getting started guides.  
+    *   [Getting Started with WebView2 in Windows Forms][Webview2GettingstartedWinforms]  
+    *   [Getting Started with WebView2 in WPF][Webview2GettingstartedWpf]  
+1.  Set the initial navigation of the WebView2 control to `geolocater.html`.  
+    
+    ```csharp
+    webView.CoreWebView2.Navigate(@"C:\{PATH TO FILE}\geolocater.html");
+    ```  
+    
+1.  Ensure the `geolocator.html` file displays in your WebView2 control app.  
+    
+    :::image type="complex" source="./media/initial-geolocate.png" alt-text="Display the geolocater.html file in your WebView2 control app" lightbox="./media/initial-geolocate.png":::
+       Display the `geolocater.html` file in your WebView2 control app  
+    :::image-end:::  
+    
+## Step 3: Install the DevToolsProtocolHelper NuGet package  
+
+Use NuGet to download `Microsoft.Web.WebView2.DevToolsProtocolExtension`.  To install the package, complete the following actions.  
+
+1.  Open Visual Studio.  
+1.  Choose **Project** > **Manage NuGet Packages** > **Browse** > **Include pre-release**.
+1.  Type `Microsoft.Web.WebView2.DevToolsProtocolExtension` and choose **Microsoft.Web.WebView2.DevToolsProtocolExtension** > **Install**.  
+    
+> [!NOTE] 
+> Ensure you also have the WebView2 NuGet Package named `Microsoft.Web.WebView2` installed.  
+
+:::image type="complex" source="./media/cdpnuget.png" alt-text="Ensure Microsoft.Web.WebView2.DevToolsProtocolExtension displays in the Visual Studio NuGet Package Manager" lightbox="./media/cdpnuget.png":::
+   Ensure **Microsoft.Web.WebView2.DevToolsProtocolExtension** displays in the Visual Studio NuGet Package Manager  
+:::image-end:::  
+
+## Step 4: Use DevTools Protocol Helper  
+
+1.  Add the `DevToolsProtocolExtension` and `CoreWebView2` namespaces to your project.
+    
+    ```csharp
+    using Microsoft.Web.WebView2.Core;
+    using Microsoft.Web.WebView2.Core.DevToolsProtocolExtension;
+    ```  
+    
+1.  Run the `GetDevToolsProtocolHelper()` method to instantiate the `DevToolsProtocolHelper` object, and navigate to `geolocater.html`.
+    
+    ```csharp
+    async void InitializeAsync()
+    {
+        await webView.EnsureCoreWebView2Async(null);
+        DevToolsProtocolHelper helper = webView.CoreWebView2.GetDevToolsProtocolHelper(); 
+        
+        webView.CoreWebView2.Navigate(@"C:\{PATH TO FILE}\geolocater.html");
+    }
+    ```  
+    
+1.  Run the [setGeoLocationOverrideAsync][setGeoLocationOverrideAsync] method.  For more information, navigate to [method-setGeolocationOverride][GithubChromedevtoolsDevtoolsProtocolTotEmulationMethodSetgeolocationOverride].  
```

Comment:
Is the `method-` prefix required?

Reply:
@pagoe-msft 
The name of the linked article includes the `method-` prefix.